docs/native/src/faq/filetypes.rst

```diff
@@ -0,0 +1,16 @@
+What file types are supported in |Office iOS|?
+============================================== 
+
+PPT
+   @"pptx", @"ppsx", @"ppt", @"pps", @"pptm", @"potm", @"potx", @"ppsm", @"pot", @"thmx", @"odp"
```

Comment:
What's the intention of the `@` sign here? It might be easier to use a bullet list. I also suggest dividing the formats into modern vs. legacy directly under each app. Then you can get rid of the 'e.g. docx' stuff at the bottom.